dbsync_daemon.py

```diff
@@ -23,6 +23,7 @@ def main():
     parser.add_argument("config_file", nargs="?", default="config.yaml", help="Path to file with configuration. Default value is config.yaml in current working directory.")
     parser.add_argument("--skip-init", action="store_true", help="Skip DB sync init step to make the tool start faster. It is not recommend to use it unless you are really sure you can skip the initial sanity checks.")
     parser.add_argument("--single-run", action="store_true", help="Run just once performing single pull and push operation, instead of running in infinite loop.")
+    parser.add_argument("--force", action="store_true", help="Force removing working directory and schemas from DB to initialize from scratch.")
```

Comment:
Thinking about this, can we rename the flag to `--force-init` ? just to make sure it is better understood what it does...